datastream-server/src/main/java/com/linkedin/datastream/server/zk/ZkAdapter.java

```diff
@@ -1434,6 +1434,13 @@ public static String parseHostnameFromZkInstance(String instance) {
     return instance.substring(0, instance.lastIndexOf('-'));
   }
 
+  /**
+   * Parse the Zk instance (ex. hostname-0000) into liveinstance
+   */
+  public static String parseLiveInstanceFromZkInstance(String instance) {
```

Comment:
nit: rename "String instance" to  "String owner"?

Reply:
Keeping this method generic since it is zk instance, which is used in lock ownership as well.